cvat/settings/development.py

```diff
@@ -9,6 +9,15 @@
 
 INSTALLED_APPS += [
     'django_extensions',
+    'corsheaders',
```

Comment:
Now I see the problem. I believe corsheaders is the right library to resolve the issue.